aldryn_jobs/migrations/0001_initial.py

```diff
@@ -2,16 +2,19 @@
 from __future__ import unicode_literals
 
 from django.db import models, migrations
-import aldryn_jobs.models
-import cms.models.fields
 from django.conf import settings
+import app_data.fields
 import djangocms_text_ckeditor.fields
+import aldryn_jobs.models
+import sortedm2m.fields
+import cms.models.fields
 
 
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('cms', '0003_auto_20140926_2347'),
+        ('cms', '0012_auto_20150607_2207'),
```

Comment:
Isn't this would be broken for cms 3.0 since there is not 0012 migration (last one is 0003 for cms 3.0)?